roles/mistral/tasks/sync.yml

```diff
@@ -1,5 +1,5 @@
 - name: Sync database
-  shell: /opt/openstack/mistral/.venv/bin/python ./tools/sync_db.py --config-file /etc/mistral/mistral.conf
+  shell: /opt/openstack/mistral/.venv/bin/python ./tools/sync_db.py --config-file /etc/mistral/mistral.conf && touch /etc/mistral/database_setup.lock
```

Comment:
Yeah, that's what I missed here https://github.com/armab/ansible-st2/commit/ce1177a17abafd0e681c57a4e7333c0e7c2f5455, good catch.

We can add `sudo: yes` for this task or go with `/opt/openstack/mistral/database_setup.lock` here and there.